entrypoint.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-if [ $# -ne 1 ]; then
-	echo Usage: $0 docs_path
+if [ $# -ne 2 ]; then
```

Comment:
GitHub will pass us the default value?

Reply:
I'm pretty sure, I've made a similar action that seems to work that way.